.github/workflows/test.yml

```
@@ -28,3 +28,9 @@ jobs:
       - name: Lint
         run: |
           pnpm run lint
+
+      - name: Dry run
```

（細かいけど）

他のnameと合わせて目的を書く方が良さそう（Dry runは手法）
```suggestion
      - name: Test
```

こうでも？
```suggestion
      - name: Test Dry Run
```